templates/tfengine/components/cicd/manual/configs/tf-plan.yaml

```diff
@@ -42,14 +42,14 @@ steps:
 
   - name: "gcr.io/cloud-foundation-cicd/cft/developer-tools@sha256:b007041726d4f2d7e5270b8fcfc47656850242186825611b33ee97b3867d74ed"
     entrypoint: bash
-    args: ["../cicd/configs/terragrunt.sh", "plan-all"]
+    args: ["../cicd/configs/terragrunt.sh", "plan-all", "-out='plan.tfplan'"]
```

Comment:
terragrunt.sh's plan-all actually includes -out=plan.tfplan as a flag, doesn't that work for you?

Reply:
Oh, you're right, I didn't notice this was running a .sh script and not terragrunt directly. Thank you, I'll remove this.